types/src/on_chain_config/timed_features.rs

```diff
@@ -58,6 +60,8 @@ impl TimedFeatureFlag {
         match (self, chain_id) {
             (DisableInvariantViolationCheckInSwapLoc, TESTNET) => NOT_YET_SPECIFIED,
             (DisableInvariantViolationCheckInSwapLoc, MAINNET) => NOT_YET_SPECIFIED,
+            (LimitTypeTagSize, TESTNET) => 1689966000000,
```

Comment:
I think you need to remove these

Reply:
This is historic right? So this is to the best of the our knowledge what was final release of v1.5

Reply:
In that case maybe we can include a human readable timestamp